socorro/scripts/fetch_crashids.py

```diff
@@ -177,17 +183,31 @@ def main(argv=None):
     params['_columns'] = 'uuid'
 
     # Override with date if specified
-    datestamp = args.date
-    if 'date' not in params or datestamp != 'yesterday':
-        if datestamp == 'yesterday':
+    if 'date' not in params or args.date:
+        datestamp = args.date or 'yesterday'
```

Comment:
This if check didn't work if you provided a `--url` value which had a `date` param and wanted to override the date with `yesterday`. I fixed it so now it should work right.